src/main/java/neqsim/processSimulation/processEquipment/ProcessEquipmentBaseClass.java

```diff
@@ -31,6 +31,7 @@ public abstract class ProcessEquipmentBaseClass extends SimulationBaseClass
     ControllerDeviceInterface flowValveController = null;
     public boolean hasController = false;
     private String specification = "TP";
+    private boolean runTransient = false;
```

Comment:
should not be named the same as the function runTransient.

Reply:
Good point. Will change that.